.circleci/config.yml

```diff
@@ -179,6 +179,30 @@ jobs:
       - EM_TESTRUNNER_DETECT_TEMPFILE_LEAKS=0
       - DISPLAY=:0
       - GALLIUM_DRIVER=softpipe # TODO: use the default llvmpipe when it supports more extensions
+  test-browser-chrome:
+    <<: *defaults
+    working_directory: ~/
+    environment:
+      - EMSCRIPTEN_BROWSER=xvfb-run $HOME/chrome-linux/chrome --no-first-run -start-maximized
```

Comment:
If you want to test with pthreads you'll also need some more flags. I have `--js-flags=--experimental-wasm-threads --enable-features=WebAssembly,SharedArrayBuffer --disable-features=WebAssemblyTrapHandler` but I'm not 100% sure all of those are actually needed anymore. Also it's probably worth adding `--user-data-dir`

Reply:
Do need those even if this is a tip of tree chrome build?

I'm not sure we need user-data-dir since we are in an empty container.  Might as well use the default no?

Reply:
I think SharedArrayBuffer is on by default on all desktop platforms now (right @binji ?) so maybe we don't need `--enable-features=SharedArrayBuffer`. The --enable-features=WebAssembly was to enable PostMessage of wasm modules, so maybe that's now subsumed by --experimental-wasm-threads. (Wasm threads is not on by default yet though so we do need that). Based on recent discussion on the V8 list, I suspect we do still need to disable WebAssemblyTrapHandler too?